__tests__/app.test.js

```diff
@@ -454,3 +454,47 @@ test("POST: 404 with Not Found for non-existent article ID", () => {
       expect(response.body.msg).toBe("Bad Request")
     })
 })
+// 8==========
+test("should update an artcle votes with a positive value", (done) => {
+  request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: 5 })
+    .expect(200)
+    .end((err, res) => {
+      if (err) return done(err)
+
+      // console.log(res.body.article, "< ---------reponse")
+      // console.log(res.body.article)
+      expect(res.body.article).toBeDefined()
+
+      expect(res.body.article.votes).toBeGreaterThan(0)
```

Comment:
Might be stronger to assert the actual value you want after the successful patch. Also you could assert that the whole article is returned and looks correct - along with the updated votes. Then everything is asserted in one test making you test suite DRYer